lambda_functions/ml_service/get_online_factcheck.py

```diff
@@ -6,7 +6,7 @@
 from core_layer import helper
 from core_layer.db_handler import Session
 from core_layer.handler import item_handler, entity_handler, keyphrase_handler
-import SearchFactChecks
+from . import SearchFactChecks
```

Comment:
This change of the import structure might lead to errors, when deployed. Instead of changing the `from` statement, you should add the folder to the path of the virtualenv, see guide [here](https://app.nuclino.com/DetektivKollektiv/Technical-Documentation/Getting-started-6f9723d5-2c56-4fb2-b132-ff573be5e2b3).

Reply:
Done